src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -512,6 +532,13 @@ private void equalizeIrrelevantData(
         }
     }
 
+    private void equalizeId(
+            FeedbackQuestionAttributes expectedFeedbackQuestion,
+            FeedbackQuestionAttributes actualFeedbackQuestion) {
```

Comment:
Ok, leave as it is for now. After this is merged, fit it across the board as a separate issue.